src/pages/iou/request/step/IOURequestStepReport.tsx

```diff
@@ -39,11 +42,21 @@ function IOURequestStepReport({route, transaction}: IOURequestStepReportProps) {
         Navigation.dismissModalWithReport({reportID: item.value});
     };
 
+    const removeFromReport = () => {
+        if (!transaction) {
+            return;
+        }
+        changeTransactionsReport([transaction.transactionID], CONST.REPORT.UNREPORTED_REPORT_ID);
+        Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(selfDMReportID));
```

Comment:
Same here. We don't want to navigate the user to the selfDM. We should navigate them to where they came from